java/pom.xml

```diff
@@ -640,8 +640,32 @@
             <scope>test</scope>
         </dependency>
     </dependencies>
+    <distributionManagement>
+        <site>
+            <id>sleeper</id>
+            <name>Sleeper site</name>
+            <url>file:///tmp/sleeper/site</url>
+        </site>
+    </distributionManagement>
     <reporting>
         <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-report-plugin</artifactId>
```

Comment:
Does this work combined into one plugin declaration with multiple report sets, like the Checkstyle one?

Reply:
It does! Must have missed that when configuring them. Pushed an update to merge that second plugin definition into a reportSet